_posts/2020-06-07-Human-Curation-and-Digital-Datasets.md

```diff
@@ -0,0 +1,213 @@
+---
+title: "Human Curation and Digital Datasets: A Problem in Multiple Parts"
+author: cgy
+date: 2020-06-07
+---
+
+{% assign author = site.data.authors[page.author]%}
+{{author.name}}
+
+ORCID: <a href="https://orcid.org/{{ author.orcid }}" title="{{author.name}}">{{author.orcid}}</a>
+
+One of the difficulties when creating and maintaining a human curated
+database is that humans are by nature error prone. In this article, I
+will explore some of the challenges and difficulties that the curators
+of the IrishGen database have encountered when dealing with ambiguous
+statements in the genealogies, how human error creeps in over time,
+and, finally, how [IrishGen](https://github.com/cyocum/irish-gen)
+attempts to avoid introducing new errors.
+
+This is not a new phenomenon within the area of the early Irish
+genealogies.  Even in the pre-modern period when scribes and redactors
+within the tradition were working, errors crept in over time.
+[Donnchadh Ó Corráin](http://isni.org/isni/0000000117710023) has
+suggested that early Irish scribes attempted to avoid errors by using
+a highly repetitious structure ('The Book of Ballymote: a Genealogical
+Treasure' in [Book of Ballymote: Codices Hibernenses Eximii
+II](http://www.worldcat.org/oclc/1046463513), pg. 10).  Ó Corráin also
+discusses in that chapter a few hypothesises which IrishGen is well
+placed to answer. Some of these errors were not even errors but
+motivated changes to the tradition by those who were paid to create or
+copy the genealogies in the first instance.  Thus, it is often
+difficult to know as someone who is looking at the genealogies in the
+modern era what counts as an error and what counts as a motivated
+change to the tradition.  As [Eystein
+Thanisch](https://orcid.org/0000-0003-2819-5519) stated in [his
+article]({{site.baseurl}}/2020/06/06/Manuscript-Variants-and-the-Origins-of-Cíannachta-Glinne-Geimen.html)
+the policy of IrishGen is to take all statements in the manuscript
+tradition seriously thus the curators attempt to preserve what is
+extant rather than attempting to editorialise.
+
+## Ditigal Curation and Data Governance
+
+The first point is Digital Curation of historical material is not a
+new subject.  The [Digital Duration Centre](https://www.dcc.ac.uk/)
```

Comment:
Digital Curation Centre

Reply:
I am not sure what you are questioning here as the Digital Curation Centre is a department of the University of Edinburgh.  I will flip it around so it reads "The University of Edinburgh's Digital Curation Centre" which should make it more clear.

Reply:
You call it the "Digital Duration Centre" in the article!!!

Reply:
I don't know how long this will go on for... :P I cannot believe I missed that